src/utils/issue.ts

```diff
@@ -173,7 +179,7 @@ export async function addAssignees(context: Context, issueNo: number, assignees:
 export async function getAllPullRequests(context: Context, state: "open" | "closed" | "all" = "open", username: string) {
   const { payload } = context;
   const query: RestEndpointMethodTypes["search"]["issuesAndPullRequests"]["parameters"] = {
-    q: `org:${payload.repository.owner.login} author:${username} state:${state}`,
+    q: `org:ubiquity org:ubiquity-os-marketplace org:ubiquity-os author:${username} state:${state}`,
```

Comment:
Here too

Reply:
Hardcoding these means third parties won't be able to use this plugin.

Reply:
Yeah, working on this one... 
Trying to test it in a custom org, but for some reasons it cant get the max task limit.. checking it

Reply:
Addressed